test/input.txt

```diff
@@ -140,6 +140,13 @@
   delete 1
   list
 
+##########################################################
+# test edit command
+##########################################################
+
+  # edits the 1st person
```

Comment:
Good job in adding a test case for the IO, maybe you can start trying out the JUnit test suite to familiarise yourself with it.